.github/workflows/ci.yaml

```diff
@@ -25,6 +25,7 @@ jobs:
           - IMAGE: melodic-ci
             CATKIN_LINT: true
     env:
+      CXXFLAGS: "-Werror -Wall -Wextra -Wwrite-strings -Wunreachable-code -Wpointer-arith -Wredundant-decls -Wno-deprecated-copy"
```

Comment:
Wouldn't we want `-pedantic -std=c++14` as well?
Also I would have expected issues with upstream library headers...
Are all potentially-problematic ones included as `-isystem` now?

Reply:
We never used `-pedantic` in the past and as our target compilers are gcc + clang, I personally don't mind allowing non-standard compiler extensions.